config/config.go

```diff
@@ -60,6 +63,9 @@ type Config struct {
 	XXX map[string]interface{} `yaml:",inline"`
 
 	networkReg map[string]Networks
+
+	// A wildcared user is found in config
+	HasWildcarded bool
```

Comment:
can you also move it before XXX? 
```suggestion
	HasWildcarded bool `yaml:"has_wildcarded,omitempty"`
```

Reply:
> can you also move it before XXX?

Should I move networkReg as well ?